Error executing job with overrides: []
Traceback (most recent call last):
  File "/Users/scanner1/Downloads/capdock/src/src/run.py", line 37, in <module>
    main()
  File "/Users/scanner1/anaconda3/envs/capdock/lib/python3.9/site-packages/hydra/main.py", line 94, in decorated_main
    _run_hydra(
  File "/Users/scanner1/anaconda3/envs/capdock/lib/python3.9/site-packages/hydra/_internal/utils.py", line 394, in _run_hydra
    _run_app(
  File "/Users/scanner1/anaconda3/envs/capdock/lib/python3.9/site-packages/hydra/_internal/utils.py", line 457, in _run_app
    run_and_report(
  File "/Users/scanner1/anaconda3/envs/capdock/lib/python3.9/site-packages/hydra/_internal/utils.py", line 223, in run_and_report
    raise ex
  File "/Users/scanner1/anaconda3/envs/capdock/lib/python3.9/site-packages/hydra/_internal/utils.py", line 220, in run_and_report
    return func()
  File "/Users/scanner1/anaconda3/envs/capdock/lib/python3.9/site-packages/hydra/_internal/utils.py", line 458, in <lambda>
    lambda: hydra.run(
  File "/Users/scanner1/anaconda3/envs/capdock/lib/python3.9/site-packages/hydra/_internal/hydra.py", line 132, in run
    _ = ret.return_value
  File "/Users/scanner1/anaconda3/envs/capdock/lib/python3.9/site-packages/hydra/core/utils.py", line 260, in return_value
    raise self._return_value
  File "/Users/scanner1/anaconda3/envs/capdock/lib/python3.9/site-packages/hydra/core/utils.py", line 186, in run_job
    ret.return_value = task_function(task_cfg)
  File "/Users/scanner1/Downloads/capdock/src/src/run.py", line 33, in main
    return train(config)
  File "/Users/scanner1/Downloads/capdock/src/src/train.py", line 79, in train
    trainer.fit(model=model, datamodule=datamodule, ckpt_path=config.ckpt_path)
  File "/Users/scanner1/anaconda3/envs/capdock/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 543, in fit
    call._call_and_handle_interrupt(
  File "/Users/scanner1/anaconda3/envs/capdock/lib/python3.9/site-packages/pytorch_lightning/trainer/call.py", line 44, in _call_and_handle_interrupt
    return trainer_fn(*args, **kwargs)
  File "/Users/scanner1/anaconda3/envs/capdock/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 579, in _fit_impl
    self._run(model, ckpt_path=ckpt_path)
  File "/Users/scanner1/anaconda3/envs/capdock/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 948, in _run
    call._call_setup_hook(self)  # allow user to set up LightningModule in accelerator environment
  File "/Users/scanner1/anaconda3/envs/capdock/lib/python3.9/site-packages/pytorch_lightning/trainer/call.py", line 94, in _call_setup_hook
    _call_lightning_datamodule_hook(trainer, "setup", stage=fn)
  File "/Users/scanner1/anaconda3/envs/capdock/lib/python3.9/site-packages/pytorch_lightning/trainer/call.py", line 181, in _call_lightning_datamodule_hook
    return fn(*args, **kwargs)
  File "/Users/scanner1/Downloads/capdock/src/src/utils/utils.py", line 303, in setup
    full_dataset = Tor_Dataset(self.full_list,self.data_dir)
  File "/Users/scanner1/Downloads/capdock/src/src/utils/utils.py", line 166, in __init__
    self.data = pd.read_csv(csv_file)
  File "/Users/scanner1/anaconda3/envs/capdock/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 1026, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/Users/scanner1/anaconda3/envs/capdock/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 620, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/Users/scanner1/anaconda3/envs/capdock/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 1620, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "/Users/scanner1/anaconda3/envs/capdock/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 1880, in _make_engine
    self.handles = get_handle(
  File "/Users/scanner1/anaconda3/envs/capdock/lib/python3.9/site-packages/pandas/io/common.py", line 873, in get_handle
    handle = open(
FileNotFoundError: [Errno 2] No such file or directory: '../../pdb_pre/tor_bois.txt'